message-index/index.html

```diff
@@ -5,10 +5,10 @@
 <h2>Welcome</h2>
 
 <p>
-  This site describes the various messages that can be returned by
-  GHC, the most-commonly-used Haskell implementation, including both errors and warnings. Beginning with version
-  9.6.1, GHC emits a code with the format <code>[GHC-12345]</code> for each
-  message. These codes can be looked up below for further information.
+  This site describes the various messages that can be returned by Haskell-related tools, including both errors and warnings.
+  For example GHC, the most-commonly-used Haskell implementation, started emiting a code with the format <code>[GHC-12345]</code> for some of its
+  messages since version 9.6.1.
```

Comment:
```suggestion
  messages in version 9.6.1.
```
To match "started". Alternatively, it could read "GHC has emitted a code with the format ... since version 9.6.1."